README.md

```diff
@@ -1,13 +1,13 @@
-# LibVMI Website
+## LibVMI Website
```

Comment:
Why do you want to add this extra `#` here? I think the main heading should remain as it is.